### ISSUE TITLE
2sxc api returns wrong language

### ISSUE THREAD

**Issue by josip8** [2sic/2sxc#2262]
**I'm submitting a ...** 

[x] bug report

**...about**   
[x] JS development
[x] APIs like REST

**Current behavior**
I have a DNN site with Croatian language as primary and English as secondary
When fetching 2sxc data over API from content it will always return Croatian language (no matter if I specify I want english), and if I fetch from query It will return English even though it is Cro version of page

**Minimal reproduction of the problem with instructions**
Here is js code: 
```
  const sxc = $2sxc(@Dnn.Module.ModuleID);
  const contentHR = await sxc.webApi.get('app/auto/content/Test');
  const contentEN = await sxc.webApi.get('/en-us/api/2sxc/app/auto/content/Test');
  const queryHR = await sxc.webApi.get('app/auto/query/Test/Default');
  const queryEN = await sxc.webApi.get('/en-us/api/2sxc/app/auto/query/Test/Default');

  document.querySelector('#sxc-data').innerHTML = `
    <p>Content - HR: ${contentHR[0].Test}</p>
    <p>Content - EN: ${contentEN[0].Test}</p>
    <p>Query - HR: ${queryHR.Default[0].Test}</p>
    <p>Query - EN: ${queryEN.Default[0].Test}</p>
  `;
```
And here is what I get:
![image](https://user-images.githubusercontent.com/26368640/100444579-edacd580-30ab-11eb-92ad-d931a35b9a4c.png)

Test app:
[2sxcApp_Test_0.0.1.zip](https://github.com/2sic/2sxc/files/5607600/2sxcApp_Test_0.0.1.zip)

* **2sxc version(s):** 11.10.01 and 11.07.01
* **Browser:** [all | Chrome XX | Firefox XX | Edge XX | IE XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari ]
* **DNN:** 9.8.0
* **Language:** [any/all | English | German | other ]

**Comment by iJungleboy** [comment]
This is odd - and not as it should be. We'll check within a few days, thx for this issue.

**Comment by iJungleboy** [comment]
I think I found the reason, and I'm trying to find a fix.

It seems that DNN sets a cookie with a language, and then that language is used in all api-requests, no matter what the url says. 

I'll try to work around this, to prefer the url-language to the cookie language, which should be how it's done.

**Comment by iJungleboy** [comment]
It was caused by DNN. I have a fix in 11.11, but I still must do some checks to verify it doesn't cause unexpected side-effects. 

Basically I'm now using the site-alias language which is more reliable than the CurrentThread language, but I must be sure that it doesn't have funny effects...
